t/geturl_connectivity.pl.in

```diff
@@ -55,35 +58,52 @@ sub run_httpd {
 
 # Note: IPv* client to IPv6 server is not expected to work unless $globals{'ipv6'} is true.
 my @test_cases = (
-    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 1 },
```

Comment:
You're missing a couple of cases:

```perl
    { ipv6_opt => 0, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 1 },
    { ipv6_opt => 0, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 1 },
```

Reply:
Thanks, not sure how that happened.  Added now.